src/Testcontainers.Pulsar/PulsarBuilder.cs

```diff
@@ -4,15 +4,15 @@ namespace Testcontainers.Pulsar;
 [PublicAPI]
 public sealed class PulsarBuilder : ContainerBuilder<PulsarBuilder, PulsarContainer, PulsarConfiguration>
 {
-    public const string PulsarImage = "apachepulsar/pulsar:3.0.6";
+    public const string PulsarImage = "apachepulsar/pulsar:3.0.8";
```

Comment:
Do the changes require bumping the patch version?

Reply:
> Do the changes require bumping the patch version?

It's not strictly required. But I would advise we do bump the patch version. It's also [backwards compatible](https://pulsar.apache.org/contribute/release-policy/#compatibility-between-releases) so it won't create any issues.